newdevtech/newdev_api
# #166276523 implement email notification on signup

#### What does this PR do?
Its adds the notification functionality for sending emails

#### Description of Task to be completed?
- add email notification
- add verify account template
- add email template
- add tests

#### How should this be manually tested?
- Checkout branch
- run  `npm install`
- run `npm run dev`
- attempt to register a user with a valid email address
- You should get an email with a verify account button

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[#166276523](https://www.pivotaltracker.com/story/show/166276523)

#### Screenshots (if appropriate)
N/A
#### Questions:
N/A
